packages/next/next-server/server/render.tsx

```
@@ -283,6 +285,12 @@ export async function renderToHTML(
     throw new Error(SPR_GET_INITIAL_PROPS_CONFLICT + ` ${pathname}`)
   }
 
+  if (!!unstable_getStaticParams && !isSpr) {
+    console.error(
```

Let's throw a real error, the console might go unnoticed.